bindings/python/ParametersHandler.cpp

```diff
@@ -0,0 +1,265 @@
+//    }
+//    bool getParameter(const std::string& parameterName, bool& parameter) const override
+//    {
+//        PYBIND11_OVERLOAD_PURE(bool, IParametersHandler, getParameter, parameterName, parameter);
+//    }
+//    // TODO: other
+//    void setParameter(const std::string& parameterName, const int& parameter) override
+//    {
+//        PYBIND11_OVERLOAD_PURE(void, IParametersHandler, setParameter, parameterName, parameter);
+//    }
+//    void setParameter(const std::string& parameterName, const double& parameter) override
+//    {
+//        PYBIND11_OVERLOAD_PURE(void, IParametersHandler, setParameter, parameterName, parameter);
+//    }
+//    void setParameter(const std::string& parameterName, const std::string& parameter) override
+//    {
+//        PYBIND11_OVERLOAD_PURE(void, IParametersHandler, setParameter, parameterName, parameter);
+//    }
+//    void setParameter(const std::string& parameterName, const bool& parameter) override
+//    {
+//        PYBIND11_OVERLOAD_PURE(void, IParametersHandler, setParameter, parameterName, parameter);
+//    }
+//    // TODO: other
+//};
+
+// class PyStdParameterHandler : public StdImplementation
+//{
+// public:
+//    using StdImplementation::StdImplementation;
+//    bool getParameter(const std::string& parameterName, int& parameter) const override
+//    {
+//        PYBIND11_OVERRIDE(bool, StdImplementation, getParameter, parameterName, parameter);
+//    }
+//    bool getParameter(const std::string& parameterName, double& parameter) const override
+//    {
+//        PYBIND11_OVERRIDE(bool, IParametersHandler, getParameter, parameterName, parameter);
+//    }
+//    bool getParameter(const std::string& parameterName, std::string& parameter) const override
+//    {
+//        PYBIND11_OVERRIDE(bool, IParametersHandler, getParameter, parameterName, parameter);
+//    }
+//    bool getParameter(const std::string& parameterName, bool& parameter) const override
+//    {
+//        PYBIND11_OVERRIDE(bool, IParametersHandler, getParameter, parameterName, parameter);
+//    }
+//    // TODO: other
+//    void setParameter(const std::string& parameterName, const int& parameter) override
+//    {
+//        PYBIND11_OVERRIDE(void, IParametersHandler, setParameter, parameterName, parameter);
+//    }
+//    void setParameter(const std::string& parameterName, const double& parameter) override
+//    {
+//        PYBIND11_OVERRIDE(void, IParametersHandler, setParameter, parameterName, parameter);
+//    }
+//    void setParameter(const std::string& parameterName, const std::string& parameter) override
+//    {
+//        PYBIND11_OVERRIDE(void, IParametersHandler, setParameter, parameterName, parameter);
+//    }
+//    void setParameter(const std::string& parameterName, const bool& parameter) override
+//    {
+//        PYBIND11_OVERRIDE(void, IParametersHandler, setParameter, parameterName, parameter);
+//    }
+//    // TODO: other
+//};
```

Comment:
If this is not needed, could you remove it?

Reply:
Woops, leftover due to rebase, thanks!

Reply:
Done in https://github.com/dic-iit/bipedal-locomotion-framework/compare/41c4270dc60602e8ba4b91a96d822ec431833ae5..5f652373b9f0be94eb827de612382ab01498014f